### DISCUSSION TITLE
Allow indent-level in config to be configurable

### DISCUSSION THREAD

**Discussion by jbergstroem** [ivanhofer/typesafe-i18n#182] [Ideas]
Perhaps silly, but I came across an issue when renovatebot bumped the dependency (which invokes a regeneration of the schema file/files) only to find itself failing the lint job since the indentlevel is different than what typesafe uses.

Should it be a config option? I'm on the fence.

**Comment by ivanhofer** [comment]
all generated files should contain an `eslint-ignore` comment. If you are using another linter, you can configure this via the [`banner`-option](https://github.com/ivanhofer/typesafe-i18n#banner).

**Reply by jbergstroem** [reply]
I do ignore the others, I guess I can add this too..

**Reply by ivanhofer** [reply]
isn't the `eslint-ignore` comment working for you?

**Reply by jbergstroem** [reply]
I disabled it since I pass a different config to `eslint` to just ignore processing the file altogether.

**Reply by ivanhofer** [reply]
maybe https://github.com/ivanhofer/typesafe-i18n/discussions/185 could be interesting for you

**Reply by jbergstroem** [reply]
Thanks for keeping me in the loop. I went ahead and ignored all files as my solution for now.

(my use-case: I use renovatebot to manage dependencies. If typesafe is updated I also regenerate the i18n files; but my CI linter would fail since I use a different indent level. Running prettier as part of a post-update job in renovate forces me to also install all dependencies and I wasn't too fond of continuously adding overhead)

**Reply by ivanhofer** [reply]
ignoring the auto-generated files was also my first intention, since e. g. gql-codegen also outputs a badly formatted file you would need to ignore.
Maybe I need to also add a prettier-ignore comment to the top of the file.
